src/hooks/useNavigateWithLocationCheck.tsx

```
@@ -0,0 +1,73 @@
+
+  const navigateWithLocationCheck = useCallback(
+    async ({
+      targetLocation,
+      address,
+      type,
+      onNavigate,
+    }: NavigateWithLocationCheckParams) => {
+      // 거리 체크
+      const {isWithin100m} = await checkDistanceFromCurrentLocation(
+        targetLocation,
+      );
```

ESLint

[error] Replace `·await·checkDistanceFromCurrentLocation(⏎········targetLocation,⏎······` with `⏎········await·checkDistanceFromCurrentLocation(targetLocation`

(prettier/prettier)